web/src/buddy_data.js

```diff
@@ -121,7 +121,8 @@ export function user_last_seen_time_status(user_id) {
         // delete this code when we finish rewriting the presence API.
         return $t({defaultMessage: "Active more than 2 weeks ago"});
     }
-    return timerender.last_seen_status_from_date(last_active_date);
+    const last_seen = timerender.last_seen_status_from_date(last_active_date);
+    return $t({defaultMessage: "Active {last_seen}"}, {last_seen});
```

Comment:
This doesn't work from either a capitalization or internationalization standpoint; we'll end up with "Active Just now", and also the sentence "Active just now' might use a different tense or something for the "active" part depending on what else is happening. 

I think the right fix is probably to instead make Recent Topics use a different function for this stuff than `last_seen_status_from_date` that doesn't have "active" in included in it; they can just be parallel and similar.

Reply:
Yeah, now that I think about it, I don't think "Active just now" sounds correct for how we are using it currently. I'm thinking of "Active recently" which fits the context better as the user had gone off the app recently for less than 2 minutes.

Reply:
After CZO discussion, we've decided to remove that phrase entirely.